react/features/toolbox/components/native/styles.js

```diff
@@ -133,6 +133,26 @@ ColorSchemeRegistry.register('Toolbox', {
         backgroundColor: BaseTheme.palette.ui13
     },
 
+    hangupMenuContainer: {
+        marginHorizontal: BaseTheme.spacing[2],
+        marginVertical: BaseTheme.spacing[2]
+    },
+
+    hangupButton: {
+        flex: 1,
+        marginHorizontal: BaseTheme.spacing[2],
+        marginVertical: BaseTheme.spacing[2]
+    },
+
+    hangupMenuButtonStyles: {
+        iconStyle: whiteToolbarButtonIcon,
+        style: {
+            ...toolbarButton,
+            backgroundColor: schemeColor('hangupMenu')
```

Comment:
Please avoid using `schemeColor`, we are phasing that out. Use the design tokens, we now have some genefic Buttons and a destructive flag which sets the right color, for example.

Reply:
Got obsolete and has been removed.